app/build.gradle

```diff
@@ -50,6 +52,9 @@ model {
         create("all") {
         }
     }
+    android.lintOptions {
```

Comment:
Lint rules got more strict, with options turned off, you can see those errors, but won't prevent you from building the application. I was able to run the app with previous gradle version (2.9), with 2.10 it failed due to Lint results.